crates/vhost/CHANGELOG.md

```diff
@@ -11,6 +11,9 @@
 
 ## [0.8.1]
 
+### Changed
```

Comment:
We already release 0.8.1, these changes should go under the `## [Unreleased]` section